From 5a979c33e4859d07660fad54d253c91202cf2a37 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?S=C3=B6ren=20Tempel?= <soeren+git@soeren-tempel.net>
Date: Wed, 10 Feb 2016 23:29:04 +0100
Subject: [PATCH 3/4] Change the default mdev path to /sbin/mdev

You can set it using the -r command line flag but I prefer it when
things work out of the box (the flag is nice to have non the less).
---
 nldev.8 | 2 +-
 nldev.c | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/nldev.8 b/nldev.8
index b1c18db..45c307b 100644
--- a/nldev.8
+++ b/nldev.8
@@ -56,7 +56,7 @@ Only show netlink messages from libudev.
 .
 .It Fl r Ar runpath
 This option specifies the runpath for the helper that is launched
-on every received netlink event (default: /bin/mdev).
+on every received netlink event (default: /sbin/mdev).
 .El
 .
 .Sh AUTHORS
diff --git a/nldev.c b/nldev.c
index deabf10..a1d529a 100644
--- a/nldev.c
+++ b/nldev.c
@@ -201,7 +201,7 @@ main(int argc, char *argv[])
 	showkernel = 1;
 	showudev = 1;
 	subsystem = NULL;
-	runpath = "/bin/mdev";
+	runpath = "/sbin/mdev";
 
 	origpath = getenv("PATH");
 	if (!origpath || strlen(origpath) <= 0)
-- 
2.7.2

